libs/utils/env.py

```diff
@@ -441,6 +448,8 @@ def _init_target(self, force = False):
 
         elif self.conf['board'] != 'UNKNOWN':
             # Initilize from platform descriptor (if available)
+            self._log.error('Warning: UNKNOWN board type detected!!')
```

Comment:
The reason is to give a easier life for the developers who want to add new board support in LISA.
Without the explicit logging information for the "UNKNOWN" board type, it takes longer time for new developers to find out the code here.

This is one reason why customers complain the LISA is not so friendly.